modules/UI/videolayout/VideoLayout.js

```diff
@@ -1155,6 +1153,27 @@ var VideoLayout = {
      */
     getRemoteVideosCount() {
         return Object.keys(remoteVideos).length;
+    },
+    /**
+     * Handles remote control active events for a remote participant.
+     *
+     * @param {JitsiParticipant} participant - The remote participant.
```

Comment:
This doc should be updated to specify just the ID is passed in. Also, the wording of the doc might be misleading because this function is used to handle remote control active-status event changes, but currently it reads that it handles events for when remote control becomes active.